src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -1001,4 +1001,72 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return OperationKind.ExpressionStatement
         End Function
     End Class
+
+    Partial Class BoundAddRemoveHandlerStatement
+        Implements IEventAssignmentExpression
```

Comment:
Since this node is a statement rather than a expression in bound tree, if I register an action on ExpressionStatement, will I get the operation that contains an IEventAssignmentExpression?